tests/src/test/java/org/eclipse/hono/tests/DownstreamMessageAssertions.java

```diff
@@ -97,7 +97,11 @@ public static void assertTelemetryMessageProperties(
                         ;
                     }
                 },
-                () -> assertMessageContainsTracingContext(msg, null));
+                () -> {
+                    if (IntegrationTestSupport.JAEGER_ENABLED) {
+                        assertMessageContainsTracingContext(msg, null);
```

Comment:
wouldn't it be easier to do this check in the *assertMessageContainsTracingContext* method itself?